examples/demo/docker-compose.yaml

```diff
@@ -11,7 +11,7 @@ services:
 
   # Zipkin
   zipkin-all-in-one:
-    image: openzipkin/zipkin:2.11.12
+    image: openzipkin/zipkin:latest
```

Comment:
Should we stick to a specific version? "latest" may be fragile.

Reply:
I think we want to go for latest in this case. We haven't being upgrading them regularly and we may take a long time to notice issues with newer versions. Notice that Jaeger was already on latest.